matching-notification-ui/src/main/resources/MatchingNotification/MatchesTable.xml

```diff
@@ -156,14 +161,17 @@ $xwiki.jsx.use('PhenoTips.Widgets')##
                       #set ($serverName = $serverID)
                     #end
                     #set ($disclaimer = $config.getProperty('disclaimer').getValue())
-                    #set ($matchesUpdateObject = $matchesRunDoc.getObject($infoClassname, 'serverName', $serverID))
-                    #if ($matchesUpdateObject)
-                      #set($endTime = $matchesUpdateObject.getValue('completedTime'))
+                    #set ($endTime = "$!matchesRunDoc.getObject($infoClassname, 'serverName', $serverID).getValue('completedTime')")
+                    #set ($endTimeInfo = $services.localization.render('phenotips.myMatches.disclaimer.noMatchRequest'))
+                    #if ($endTime &amp;&amp; "$!endTime.class" == 'class java.lang.String' &amp;&amp; "$!endTime" != "")
+                      #set ($endTimeParsed = $services.jodatime.getDateTimeFormatterForPattern('EEE MMM dd kk:mm:ss z yyyy').parseDateTime($endTime).toDate())
+                      #set ($endTimeFormatted = $services.jodatime.getDateTimeFormatterForPattern($dateFormat).print($endTimeParsed))
+                      #set ($endTimeInfo = $services.localization.render('phenotips.myMatches.disclaimer.matchesLastRun', [$endTimeFormatted]))
                     #end
                     &lt;label&gt;
                       &lt;input name="checkbox-server-id-filter" class="mme" type="checkbox" value="${escapetool.xml($serverID)}" checked&gt;
                       &lt;input type="hidden" class="disclaimer" value="${escapetool.xml($disclaimer)}"&gt;
-                      &lt;input type="hidden" class="endTime" value="${escapetool.xml($!endTime)}"&gt;
+                      &lt;input type="hidden" class="endTime #if($endTime)time-updated#else never-run#end" value="${escapetool.xml($!endTimeInfo)}"&gt;
```

Comment:
The `time-updated` css class is added when the matches were never computed, please see screenshot:

![image](https://user-images.githubusercontent.com/651980/63055300-96352700-beb3-11e9-9c17-f97eb519a6e7.png)